### PR TITLE
Add acceptance test.

### PR DESCRIPTION
**Pull request by jmcarp**
Need help from @dlapiduz to delegate a domain to the hosted zone in the right aws account.

### CODE REVIEW

File: ci/acceptance-tests.sh [hunk 1]
```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Set defaults
+TTL="${TTL:-60}"
+
+suffix="${RANDOM}"
```
Comment (afeld) [inline ci/acceptance-tests.sh]: Today I learned!

https://twitter.com/aidanfeldman/status/814869389480697857

File: ci/acceptance-tests.sh [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  --change-batch file://./create-cname.json
+
+# Wait for provision to complete
+elapsed=3600
+until [ "${elapsed}" -le 0 ]; do
+  status=$(cf service "${SERVICE_INSTANCE_NAME}" | grep "^Status: ")
+  if [[ "${status}" =~ "Status: create succeeded" ]]; then
+    updated="true"
+    break
+  elif [[ "${status}" =~ "Status: create failed" ]]; then
```
Comment (afeld) [inline ci/acceptance-tests.sh]: Maybe just check for "succeeded"/"failed"? I worry that this could be brittle if the language/punctuation changes.

**Comment by afeld** [comment]
Also, I assume this isn't `[WIP]` anymore?

**Comment by jmcarp** [comment]
@afeld: I'd be happy to port the tests to a real language, and even happier to merge a patch doing it. We usually write these tests in bash because most of them don't get this complicated, but maybe it's time to rethink that choice.

**Comment by afeld** [comment]
No, it looks really good...I just would have given up if I were trying to do it in bash 😄  I only suggest it because for more complicated things like this, another scripting language could make the logic easier to follow, give better tools for parsing output, etc.

**Pull request merged**